charts/atlantis/templates/statefulset.yaml

```
@@ -29,6 +29,7 @@ spec:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap-config.yaml") . | sha256sum }}
         checksum/repo-config: {{ include (print $.Template.BasePath "/configmap-repo-config.yaml") . | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/extra-manifests.yaml") . | sha256sum }}
```

```suggestion
        checksum/extra-manifests: {{ include (print $.Template.BasePath "/extra-manifests.yaml") . | sha256sum }}
```
use a unique name for annotations key